scripts/kind-e2e/e2e.sh

```diff
@@ -477,9 +476,12 @@ elif [[ $5 = helm ]]; then
 fi
 
 test_connection
-test_with_e2e_tests
 
-if [[ $1 = keep ]]; then
+if [[ $1 != create ]]; then
```

Comment:
I find this if clause weird. Why wouldn't you check if it's `keep` which is the case where it would be run?

Reply:
Done